docs/upgrade.md

```diff
@@ -45,6 +45,20 @@ for detail on upgrading to Marionette 3. This technique works in both Marionette
 
 The `childEvents` attribute was renamed to `childViewEvents`.
 
+Child view events now only bubble a single level to their direct parent. To
```

Comment:
So Marionette has never intended on including event bubbling beyond one level.  It was added by accident.. only for certain situations when combining layouts with collectionviews. https://github.com/marionettejs/backbone.marionette/issues/2667#issuecomment-130537382  This accidental behavior has yet to be removed.  Personally I'm hoping in the future to entirely remove the auto-proxying and have people rely on `childViewTriggers` for explicit.. way less noisy.. events.